templates/security/cve/index.html

```diff
@@ -81,7 +81,7 @@ <h2>Search</h2>
   </form>
 </section>
 
-<section class="p-strip is-shallow">
+<section class="p-strip is-shallow" style="overflow-x: inherit">
```

Comment:
I think this is not good practice. it's harder to maintain inline styles. Maybe adding a class to `cve-pattern` or maybe it would be even better to have a utility class e.g. `u-overflow-inherit` in Vanilla.

Reply:
Made the change.